From 3fe8644bf00f4fd54fcc6b2cbb79db2705c52b56 Mon Sep 17 00:00:00 2001
From: Travis Watkins <amaranth@ubuntu.com>
Date: Wed, 23 Apr 2014 16:21:26 -0500
Subject: [PATCH] Don't limit anvil stack size.

After the changes in c549609d anvils started incorrectly limiting their
inputs to single item stacks. Vanilla allows for full size stacks here and
this is useful for using materials to repair items so now we do too.
---
 src/main/java/net/minecraft/server/ContainerAnvilInventory.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/main/java/net/minecraft/server/ContainerAnvilInventory.java b/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
index 1afa6e7..6cc0d92 100644
--- a/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
+++ b/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
@@ -43,7 +43,6 @@ public class ContainerAnvilInventory extends InventorySubcontainer { // CraftBuk
     ContainerAnvilInventory(ContainerAnvil containeranvil, String s, boolean flag, int i) {
         super(s, flag, i);
         this.a = containeranvil;
-        this.setMaxStackSize(1); // CraftBukkit
     }
 
     // CraftBukkit start - override inherited maxStack from InventorySubcontainer
-- 
1.8.3.2

